test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -450,6 +452,19 @@ def setUpClass(cls):
         cls.check_users_exist()
         cls.init_servers()
         if cls.use_cur_setup:
+            test_user = pwd.getpwuid(os.getuid())[0] + '@*'
+            cls.users = {test_user: "managers",
+                         str(MGR_USER) + '@*': "managers",
+                         str(OPER_USER) + '@*': "operators"}
+            server_stat = cls.server.status(SERVER, ["managers", "operators"])
+            if len(server_stat) > 0:
+                server_stat = server_stat[0]
+            for user, role in cls.users.items():
+                if role not in server_stat or user not in server_stat[role]:
+                    attr = {role: (INCR, user)}
+                    cls.server.manager(MGR_CMD_SET, SERVER, attr, sudo=True)
+                    cls.added_users.append(user)
```

Comment:
rather than adding just user in `added_users`, you can add user + role, so you don't need 2nd class variable.